GROUP,PACKAGE,CLASS,INSTRUCTION_MISSED,INSTRUCTION_COVERED,BRANCH_MISSED,BRANCH_COVERED,LINE_MISSED,LINE_COVERED,COMPLEXITY_MISSED,COMPLEXITY_COVERED,METHOD_MISSED,METHOD_COVERED
consumer,online.caltuli.consumer.api.abuseipdb.exception,AbuseIpDbException,0,4,0,0,0,2,0,1,0,1
consumer,online.caltuli.consumer.api.abuseipdb.exception,ConfigurationLoadException,0,4,0,0,0,2,0,1,0,1
consumer,online.caltuli.consumer.api.abuseipdb.exception,JsonDoesNotDescribeIpDetailsException,0,4,0,0,0,2,0,1,0,1
consumer,online.caltuli.consumer.api.abuseipdb,AbuseIpDbClient,27,69,0,4,6,22,1,7,1,5
consumer,online.caltuli.consumer.api.abuseipdb,IpValidator,5,232,4,16,1,39,5,9,1,3
consumer,online.caltuli.consumer.api.abuseipdb,IpValidator.new TypeReference() {...},0,3,0,0,0,1,0,1,0,1
consumer,online.caltuli.consumer.dao,DaoFactory,56,0,2,0,17,0,6,0,5,0
consumer,online.caltuli.consumer.dao.exceptions,InvalidUserDetailsException,4,0,0,0,2,0,1,0,1,0
consumer,online.caltuli.consumer.dao.exceptions,DatabaseConnectionException,4,0,0,0,2,0,1,0,1,0
consumer,online.caltuli.consumer.dao.exceptions,DaoException,4,0,0,0,2,0,1,0,1,0
consumer,online.caltuli.consumer.dao.exceptions,UserDataAccessException,4,0,0,0,2,0,1,0,1,0
consumer,online.caltuli.consumer.dao.implementations,UsersDaoImpl,362,0,12,0,92,0,11,0,5,0
consumer,online.caltuli.consumer.dao.implementations,UserConnectionsDaoImpl,245,0,18,0,69,0,14,0,5,0
